CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/jotai/issues/new).
+
+For usage questions, prefer [starting a discussion](https://github.com/pmndrs/jotai/discussions/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Development
+
+If you have been assigned to fix an issue or develop a new feature, please follow these steps to get started:
```

Comment:
Maybe, instead of asking to follow the steps, showing them as an example?
Also, assignment might not be necessary to start development.

Reply:
Agreed, mentioning assignment can scare off new contributors (like myself)

Reply:
yeah, we should have an example folder in the repo.

Reply:
Perhaps:
`If you would like to contribute by fixing an open issue or developing a new feature you can use this suggested workflow:`